### PR TITLE
rename npm run start:dev to npm run serve:dev lub npm run dev whatever (effort: 1) #67

### LINKED ISSUE #67: rename npm run start:dev to npm run serve:dev lub npm run dev whatever (effort: 1)

### PR DESCRIPTION
**Pull request by ibednorz**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -31,8 +31,8 @@
     "node": "15.4.0"
   },
   "scripts": {
-    "start": "node index.js",
-    "start:dev": "nodemon index.js",
+    "dev": "node index.js",
```
Comment (pdybowski) [inline package.json]: this one must be start

File: package.json [hunk 2]
```diff
@@ -31,8 +31,8 @@
...
+    "dev:watch": "nodemon index.js",
```
Comment (pdybowski) [inline package.json]: just dev

**Review by pdybowski — COMMENTED** [review]
fix

File: package.json (hunk 2 above)
Comment (bartoszbialecki) [reply package.json]: yes, only dev and dev should be nodemon index.js and start must stay

**Pull request merged**